org-gtasks.el

```diff
@@ -339,7 +339,7 @@
                  (notes (replace-regexp-in-string
                          org-gtasks-links-drawer-re "" notes))
                  ;; Strip leading and trailing newlines in notes
-                 (notes (replace-regexp-in-string "\\`\n+" "" notes))
+                 (notes (string-trim notes))
                  (notes (replace-regexp-in-string "\n+\\'" "" notes))
```

Comment:
you don't need this line anymore:
`(notes (string-trim notes))`
replace
```
(notes (replace-regexp-in-string "\\`\n+" "" notes))
(notes (replace-regexp-in-string "\n+\\'" "" notes))
```

Reply:
Thanks for the squash but you still have in your commit:
`(notes (replace-regexp-in-string "\n+\\'" "" notes))`
Please remove this line, amend this patch and that will be perfect :)